doc/release/master/refactor_cleanUpNwcNws.md

```diff
@@ -0,0 +1,24 @@
+refactor_cleanUpNwcNws {#master}
+-------------------
+
+### Devices
+
+#### `yarp::dev`
+
+* Refactor and cleanup of all the currently available nws and nws (see [#2441](https://github.com/robotology/yarp/discussions/2441) for clarifications about nws/nws architecture).
+* The modified nws/nwc are the following:
+  * ControlBoard_nws_ros
+  * ControlBoard_nws_yarp
+  * rgbdSensor_nws_ros
+  * rgbdSensor_nws_yarp
+  * RGBDToPointCloudSensor_nws_ros
+  * rangefinder2D_nws_ros
+  * rangefinder2D_nws_yarp
+  * FrameGrabber_nwc_yarp
+  * frameGrabber_nws_ros
+  * frameGrabber_nws_yarp
```

Comment:
Is it intended to have `frameGrabber_nwc_yarp` and `FrameGrabber_nws_yarp`? Or it is just a typo

Reply:
It's just a typo. Thanks for pointing it out!